V_QuadMetric.cpp

```diff
@@ -724,22 +726,22 @@ double quad_warpage(int /*num_nodes*/, const double coordinates[][3])
     return (double)VERDICT_DBL_MIN;
   }
 
-  double warpage = std::pow(
-    std::min(corner_normals[0] % corner_normals[2], corner_normals[1] % corner_normals[3]), 3);
+  double warpage = fmin(corner_normals[0] % corner_normals[2], corner_normals[1] % corner_normals[3]);
+  warpage = warpage * warpage * warpage;
```

Comment:
The c pow function is much slower for integer exponents than manually multiplying. Is this change acceptable or would you like me to write a cube function?